src/client/components/calcAvg.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+const [strengthAvg, setStrengthAvg] = useState(0)
+const [companionAvg, setCompanionAvg] = useState(0)
+const [designAvg, setDesignAvg] = useState(0)
+const [ratingsArray, setRatingsArray] = useState([])
+
+const fetchRatings = () => {
+    const ratingUrl = "http://localhost:4000/ratings"
```

Comment:
now that Theunis has implemented the env url we should go back and make sure we havent missed any hardcoded routes

Reply:
Good plan

Reply:
👍🏿